src/webui/src/components/trial-detail/TableList.tsx

```diff
@@ -373,136 +388,132 @@ class TableList extends React.Component<TableListProps, TableListState> {
                 parameterStr.push(`${value} (search space)`);
             });
         }
+        let showTitle = COLUMNPro; // eslint-disable-line @typescript-eslint/no-unused-vars
         showTitle = COLUMNPro.concat(parameterStr);
 
         // only succeed trials have final keys
         if (tableSource.filter(record => record.status === 'SUCCEEDED').length >= 1) {
             const temp = tableSource.filter(record => record.status === 'SUCCEEDED')[0].accuracy;
```

Comment:
`temp`: better to give it a meaningful name.

Reply:
Ok, thanks